docs/api/ApiIndex.vue

```diff
@@ -101,7 +101,11 @@ onUnmounted(() => window.removeEventListener('keydown', apiSearchFocusHandler));
           </h3>
           <ul>
             <li v-for="h of item.headers" :key="h.anchor">
-              <a :href="item.link + '#' + slugify(h.anchor)">{{ h.text }}</a>
+              <a
+                :href="item.link + '#' + slugify(h.anchor)"
+                :class="h.deprecated ? 'deprecated' : ''"
```

Comment:
Should we simplify this to:

```suggestion
                :class="{ deprecated: h.deprecated }"
```

Reply:
yes, we should